src/Orleans.Core/Streams/Internal/IInternalAsyncObservable.cs

```diff
@@ -18,7 +23,7 @@ internal interface IInternalAsyncObservable<T> : IAsyncObservable<T>
     }
 
         
-    internal interface IInternalAsyncBatchObserver<in T> : IAsyncBatchObserver<T>
+    internal interface IInternalAsyncBatchObserver<T> : IAsyncObserver<T>
```

Comment:
I don't understand what this interface is used for anymore...

Reply:
yeah, it's legacy junk we've not clear out yet. IMO. :/